### ISSUE TITLE
BatchNorm mean/variance EMA is not added into tf.GraphKeys.MODEL_VARIABLES while using SyncBN

### ISSUE THREAD

**Issue by elmirador** [tensorpack/tensorpack#1270]
https://github.com/tensorpack/tensorpack/blob/master/tensorpack/models/batch_norm.py#L282-L368

Not a big issue at all, but it'll be nice if the behavior is unified since it does add them into `MODEL_VARIABLES` under `sync_statistics=None`:
https://github.com/tensorpack/tensorpack/blob/master/tensorpack/models/batch_norm.py#L257-L260
